src/Serilog.Sinks.IOFile/project.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The file sink for Serilog - Simple .NET logging with fully-structured events",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "file", "io" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": { "target": "project" }
+  },
+  "compilationOptions": {
+    "keyFile": "../../assets/Serilog.snk"
+  },
+  "frameworks": {
+    "net45": {
+    },
+    "dotnet5.1": {
+      "dependencies": {
+        "System.IO": "4.0.11-beta-23516",
+        "System.IO.FileSystem":  "4.0.0",
+        "System.IO.FileSystem.Primitives": "4.0.0"
+      }
+    },
+    "dotnet5.4": {
+      "dependencies": {
+        "System.IO": "4.0.11-beta-23516",
+        "System.IO.FileSystem.Primitives": "4.0.1-beta-23516",
+        "runtime.osx.10.10.System.IO.FileSystem": "4.0.1-beta-23516",
```

Comment:
Are these needed? I guess these can be moved into the `runtimes` node, once that's supported:

```json
"runtimes": {
    "win7-x86": { },
    "win7-x64": { },
    "win10-x86-aot": { },
    "osx.10.11-x64": { },
    "centos.7-x64": { },
}
```